dt/uni04delta/edpm/nodeset/nova_ceph.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ceph-nova
+data:
+  03-ceph-nova.conf: _replaced_
+
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: nova-custom-ceph
+spec:
+  label: dataplane-deployment-nova-custom-ceph
```

Comment:
missing caCerts: combined-ca-bundle

can also add containerImageFields as in https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml, not related to the issue but good to keep in sync.